impl/src/main/java/org/ehcache/internal/persistence/DefaultLocalPersistenceService.java

```diff
@@ -77,12 +79,13 @@ public synchronized void stop() {
         // passes on windows
         rw.close();
         if (!lockFile.delete()) {
-          // todo log something?;
+          LOGGER.info("Lock file was not deleted {}.", lockFile.getPath());
```

Comment:
I agree with using WARN here, but the dev-guidelines specifically discourage using WARN